log_queries.py

```diff
@@ -81,3 +88,28 @@ def log_query(service, question: str, answer: str, sentiment: str = "N/A", sprea
         return 1
 
     return 0
+
+@app.route("/log/query", methods=["POST"])
+def log_route():
+    request_body = request.get_json()
+    question = request_body.get("question")
+    answer = request_body.get("answer")
+    sentiment = request_body.get("sentiment")
+    request_body = request.get_json()
+    question = request_body.get("question")
+    answer = request_body.get("answer")
+    sentiment = request_body.get("sentiment")
```

Comment:
Is this duplicate code?

Reply:
Looks like it

Reply:
At least, just 100-101

Reply:
Yeah I don't know how I did that...fixed now!